user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.md

```diff
@@ -8,57 +8,53 @@ As of DataMiner Cloud Pack version 2.5.0, you can use the Admin app to manage th
+1. In the [Admin app](xref:Accessing_the_Admin_app), check whether the correct organization is mentioned in the header bar.
 
 1. If a different organization should be selected, click the organization selector ![Organization selector](~/user-guide/images/Cloud_Admin_Selector_icon.png) in the top-right corner and select the organization in the list.
 
-1. In the pane on the left, under *DataMiner Systems*, select your DataMiner System and select the *Nodes* page.
+1. In the pane on the left, under *DataMiner Systems*, expand the DataMiner System, and select the *Nodes* page.
+
+   This page lists the different nodes in the DMS, with all possible DxMs that can be deployed on each node.
 
-1. Next to your module of choice, e.g. *ModelHost*, click *Deploy* to start the automatic installation process.
+1. Next to a DxM, e.g. *ModelHost*, click *Deploy* to start the automatic installation process.
 
 ## Upgrading nodes to the latest DxM versions
 
 To upgrade the DxMs used by the nodes in a DataMiner System connected to dataminer.services:
 
-1. In the Admin app, check whether the correct organization is mentioned in the header bar.
-
-   > [!TIP]
-   > See also: [Accessing the Admin app](xref:Accessing_the_Admin_app)
+1. In the [Admin app](xref:Accessing_the_Admin_app), check whether the correct organization is mentioned in the header bar.
 
 1. If a different organization should be selected, click the organization selector ![Organization selector](~/user-guide/images/Cloud_Admin_Selector_icon.png) in the top-right corner and select the organization in the list.
 
-1. In the pane on the left, under *DataMiner Systems*, select your DataMiner System and select the *Nodes* page.
+1. In the pane on the left, under *DataMiner Systems*, expand the DataMiner System, and select the *Nodes* page.
+
+1. To upgrade all DxMs installed on a node to the latest version, click *Install latest available versions*.
 
-1. To upgrade all DxMs installed on a node to the latest version, click *Install latest available versions*. Alternatively, to upgrade specific DxMs only, click the *Upgrade* button for those DxMs.
+   Alternatively, to upgrade specific DxMs only, click the *Upgrade* button for those DxMs.
 
    > [!NOTE]
    > If a node has a higher DxM version installed than the current version, a *Downgrade* button will be displayed instead.
 
 1. To view the changes after the upgrade, click the *Refresh* button.
 
-## Removing an obsolete node from dataminer.services
-
-When a node in a DataMiner System is no longer used, you can remove it from dataminer.services in the Admin app.
+> [!NOTE]
+> If you want to quickly check whether a DMS needs DxM upgrades, on the *Organization* > *Overview* page, click the DMS name under *DataMiner Systems*. This will show a page with information on the (connection) status of the DMS and the available updates.<!-- RN 38960 -->
 
-To do so:
+## Removing an obsolete node from dataminer.services
```

Comment:
What happens when you do this for a DaaS system?

Reply:
Same as for a normal system, orchestrator will send a message again automatically about the dxms it has detected and that will repopulate the node.